components/service/glean/scripts/sdk_generator.gradle

```diff
@@ -5,6 +5,25 @@
 import org.apache.tools.ant.taskdefs.condition.Os
 
 String GLEAN_PARSER_VERSION = "0.12.0"
+String MINICONDA_VERSION = "4.5.11"
+File GLEAN_BOOTSTRAP_DIR = new File(
+    project.getGradle().gradleUserHomeDir,
+    "glean/bootstrap-${MINICONDA_VERSION}"
+)
+File GLEAN_MINICONDA_DIR = new File(
+    GLEAN_BOOTSTRAP_DIR,
+    "Miniconda3"
+)
+
+android {
+    defaultConfig {
+        buildConfigField(
+            "String",
+            "GLEAN_MINICONDA_DIR",
+            "\"" + GLEAN_MINICONDA_DIR + "\""
+        )
+    }
```

Comment:
This will be part of the public API of the library. Not necessarily problematic though.

Reply:
Yes. Do you know off-hand if there's a way of making it "for testing only"?  It really should only ever be used from running unit tests.  I figured out how to do debug vs. release, but not test vs. non-test.